Maple2Storage/Types/Metadata/MapEntityMetadata.cs

```diff
@@ -269,6 +294,42 @@ public MapPlayerSpawn(CoordS coord, CoordS rotation)
             $"MapPlayerSpawn(Coord:{Coord},Rotation:{Rotation})";
     }
 
+    [XmlType]
+    public class MapInteractActor
```

Comment:
if MapInteractActor and MapInteractMesh have the same fields we can reuse the same class.

Reply:
For now they're the same because meshes aren't implemented yet. The next PR will implement them, and (I think), it'll have different setup. This is just a stub for now.